website/redirects.next.js

```diff
@@ -137,13 +137,13 @@ module.exports = [
     permanent: true,
   },
   {
-    source: '/docs/configuration/from-1.5/:path*',
-    destination: '/docs/from-1.5/:path*',
+    source: '/docs/templates/hcl_templates/:path*',
+    destination: '//docs/templates/hcl_templates/:path*',
```

Comment:
```suggestion
    destination: '/docs/templates/hcl_templates/:path*',
```

Reply:
Thanks for catching these-- you're right it'll be easier to replace in a single search/replace commit, which is incoming!